Docs/source/running_cpp/parameters.rst

```diff
@@ -180,6 +180,29 @@ Setting up the field mesh
 * ``warpx.n_rz_azimuthal_modes`` (`integer`; 1 by default)
     When using the RZ version, this is the number of azimuthal modes.
 
+.. _running-cpp-parameters-parallelization:
+Domain Boundary Conditions
```

Comment:
@dpgrote and @ax3l May be I should add the documentation only for the BC that is supported in this PR, i.e., periodic?
So that we dont confuse users with options that are not currently supported through the interface? What do you think?
I have added it below anyway, but, as I was writing it out, I realized it may lead to confusion.

Reply:
Yes, that would be good, to only add documentation for what is supported now. The rest can be added later with the subsequent PRs.

Reply:
sounds good. I will edit the doc above, and push the changes